Composer/packages/client/src/components/CreationFlow/AzureBotDialog.tsx

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import React, { useState } from 'react';
+import formatMessage from 'format-message';
+import { FontWeights } from 'office-ui-fabric-react/lib/Styling';
+import { FontSizes } from '@uifabric/fluent-theme';
+import { PrimaryButton, DefaultButton } from 'office-ui-fabric-react/lib/Button';
+import { Link } from 'office-ui-fabric-react/lib/Link';
+import { ChoiceGroup, IChoiceGroupOption } from 'office-ui-fabric-react/lib/ChoiceGroup';
+import { DialogFooter, IDialogContentStyles } from 'office-ui-fabric-react/lib/Dialog';
+import { DialogWrapper, DialogTypes } from '@bfc/ui-shared';
+import { RouteComponentProps } from '@reach/router';
+
+import TelemetryClient from '../../telemetry/TelemetryClient';
+import { DialogCreationCopy } from '../../constants';
+
+type Props = {
+  isOpen: boolean;
+  onDismiss: () => void;
+  onJumpToOpenModal: (search?: string) => void;
+  setIsOpenCreateModal: (boolean) => void;
```

Comment:
`setIsOpenCreateModal` should be `onToggleCreateModal`. Event handlers passed as props should not start with `set*`.